ReactiveXComponent/Parser/XCApiConfigParser.cs

```diff
@@ -11,55 +11,61 @@ namespace ReactiveXComponent.Parser
 {
     public class XCApiConfigParser : IXCApiConfigParser
     {
-        private XCApiDescription _xcApiDescription;
-
-        private Dictionary<string, int> _componentCodeByComponent;
-        private Dictionary<string, int> _stateMachineCodeByStateMachine;
+        private XCApiDescription _xcApiDescription;
+        private XNamespace _xc;
         private Dictionary<string, int> _eventCodeByEvent;
         private Dictionary<TopicIdentifier, string> _publisherTopicByIdentifier;
         private Dictionary<TopicIdentifier, string> _subscriberTopicByIdentifier;
         private Dictionary<long, string> _snapshotTopicByComponent;
+        private Dictionary<string, List<StateMachineInfo>> _stateMachineInfoByComponentRepository;
```

Comment:
Choose something "Repo" "Repository" or "Nothing" and use it everywhere. I would suggest to remove Repository & Repo.

Reply:
done